components/datetime/src/provider/gregory.rs

```diff
@@ -59,10 +59,6 @@ macro_rules! symbols {
         ($name: ident { $element: ident: Option<$ty: ty>, $($tokens: tt)+ } -> ($($members:tt)*)) => {
             symbols!($name { $($tokens)* } -> (
                 $($members)*
-                #[cfg_attr(
-                    all(feature="provider_serde", not(feature="serialize_none")),
-                    serde(skip_serializing_if = "Option::is_none"))
-                ]
```

Comment:
nitpick: Leave a TODO referencing a new issue number in all the places where we want to add `skip_serializing_if` back again, to make it easy to find?

Reply:
I thought about this and decided against it: if we wish to re-add it we can visit this PR again.

The problem is that more such fields will be added in the future, so it feels like it's better to not try to mark them right now, and I'm not a _huge_ fan of leaving code breadcrumbs for stuff that is in the version history.